examples/samples/root_headerECAT.hroot

```diff
@@ -0,0 +1,61 @@
+ROOT header := 
+
```

Comment:
add a comment like `This is an example file for a ROOT file generated with GATE for a Siemens mMR-like scanner (but using an 8x9 block) and using the ECAT system as output.`